src/main/java/org/folio/circulation/domain/NoteRepresentation.java

```diff
@@ -0,0 +1,55 @@
+package org.folio.circulation.domain;
+
+import java.util.Arrays;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+public class NoteRepresentation extends JsonObject {
+  public NoteRepresentation(NoteRepresentationBuilder builder) {
+    super();
+
+    this.put("title", builder.title);
+    this.put("typeId", builder.typeId);
+    this.put("content", builder.content);
+    this.put("domain", builder.domain);
+    this.put("links", builder.links);
+  }
```

Comment:
My understanding is this is relationship is typically the reverse of this, that the builder constructs an instance of the class and the class does not know about the builder. 

Why does the note representation need to know about the builder?